README_first.ipynb

```diff
@@ -1,60 +1,160 @@
 {
```

Comment:
<p>Suggestion for title: Image generation with Stable Diffusion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>